jupyterlab_server/__init__.py

```diff
@@ -16,3 +16,11 @@
     'SETTINGS_EXTENSION',
     'WORKSPACE_EXTENSION'
 ]
+
+def _jupyter_server_extension_paths():
```

Comment:
This function has been changed to `_jupyter_server_extension_points` in jupyter_server.

It will still work with this old name, but it would be good to update here.

Reply:
Thx. Done for jupyterlab_server. I will roll that change in jupyterlab also